std-bits/table/src/main/java/org/enso/table/parsing/BaseTimeParser.java

```diff
@@ -12,22 +13,26 @@ protected interface ParseStrategy {
   }
 
   protected final DateTimeFormatter[] formatters;
+  protected final boolean[] replaceSpaces;
   protected final ParseStrategy parseStrategy;
 
   protected BaseTimeParser(String[] formats, Locale locale, ParseStrategy parseStrategy) {
     this.parseStrategy = parseStrategy;
 
     formatters = new DateTimeFormatter[formats.length];
+    replaceSpaces = new boolean[formats.length];
     for (int i = 0; i < formats.length; i++) {
-      formatters[i] = DateTimeFormatter.ofPattern(formats[i], locale);
+      formatters[i] = Time_Utils.make_formatter(formats[i], locale);
+      replaceSpaces[i] = formats[i].endsWith("_DATE_TIME");
```

Comment:
I'd prefer the logic needed to figure out if we need to replace the spaces to also reside in `Time_Utils` - I think it would be beneficial to gather all of this kind of logic in one place.